File: android/app/src/main/java/com/bensbible/app/model/BibleLocation.kt
package com.bensbible.app.model

data class BibleLocation(
    val bookName: String,
    val chapterNumber: Int,
    val verseNumber: Int? = null
) {
    val displayTitle: String get() = "$bookName $chapterNumber"

    companion object {
        val genesis1 = BibleLocation(bookName = "Genesis", chapterNumber = 1)
    }
}

File: BensBible2/Models/BibleLocation.swift
import Foundation

struct BibleLocation: Equatable, Codable {
    var bookName: String
    var chapterNumber: Int
    var verseNumber: Int?

    static let genesis1 = BibleLocation(bookName: "Genesis", chapterNumber: 1)

    var displayTitle: String {
        "\(bookName) \(chapterNumber)"
    }
}
